rocAL/rocAL/source/augmentations/geometry_augmentations/node_crop_mirror_normalize.cpp

```diff
@@ -43,42 +43,54 @@ void CropMirrorNormalizeNode::create_node()
         THROW("Uninitialized destination dimension - Invalid Crop Sizes")
 
     _crop_param->create_array(_graph);
-    _mean_vx.resize(_batch_size);
-    _std_dev_vx.resize(_batch_size);
-    for (uint i=0; i < _batch_size; i++ ) {
-         _mean_vx[i] = _mean;
-         _std_dev_vx[i] = _std_dev;
+
+    std::vector<float> mean_vx, std_dev_vx;
+    mean_vx.resize(_batch_size * 3);
+    std_dev_vx.resize(_batch_size * 3);
+    for (uint i = 0, j = 0; i < _batch_size; i++ , j += 3) {
+        mean_vx[j] = _mean[0];
```

Comment:
if input is PLN1 image this will not be valid. Please add an else for handling PLN1 inputs